common/scripted_effects/hfe_effect_pirate.txt

```diff
@@ -290,8 +293,37 @@ hfe_effect_pirate_init_country = {
 				country_remove_ethic = ethic_fanatic_pacifist
 				country_add_ethic = ethic_fanatic_militarist
 			}
+			if = {
```

Comment:
Pirates can't be Xenophiles for obvious reasons. The reason I cut the Egalitarian thing is because it was an unnecessary overwrite of Vanilla rendition files, which accomplishes nothing and changes no mechanics. Egalitarian Pirates don't do anything special.

Reply:
I do not see how pirates cant be xenophiles or egalitarians

Reply:
I discussed this in the chat rooms.

Reply:
It doesnt make my agree

Reply:
Pirates cannot be Xenophiles and Pirate Egalitarians break compatibility with most Mods and the Vanilla rendition itself. They are completely unnecessary. You always talk about redundant code and not messing with the Vanilla rendition code whenever possible.

Besides, you already allowed it in the past.

Reply:
It never happens with my pirates.

Reply:
You don't get any Pirate Egalitarians?